styles/components/_alert.scss

```diff
@@ -0,0 +1,21 @@
+.alert {
+  display: flex;
+  border-width: 2px;
+  border-style: solid;
+  border-color: none;
+  border-radius: 0.25rem;
```

Comment:
Perhaps a variable? Check with @synowa, she created a variables file.

Reply:
To avoid conflicts it will be changed after variables will be merged with develop.